src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -47,13 +47,24 @@ websocket.filter.label.regex.inverse            = Inverse
 websocket.fuzz.fail                             = Error
 websocket.fuzz.success                          = Successful
 websocket.manual_send.fail                      = Unable to send crafted message!
+websocket.manual_send.fail.unable_reopen        = It is not able to re-open Connection.
```

Comment:
`It is not possible to re-open the Connection.`
(Does connection need a capital?)